packages/react/src/components/Link/Link.stories.tsx

```diff
@@ -69,7 +69,7 @@ export const InText: StoryFn<typeof Link> = () => (
 export const LongLink: StoryFn<typeof Link> = () => (
   <Paragraph>
     <Link href='https://designsystem.no/'>
-      Dette er en linke som brekker over flere linjer
+      Dette er en link som brekker over flere linjer
```

Comment:
"lenke"?

Reply:
Hard for me to know if the person that made the Story made a typo for 'link' or 'lenke' since the word originally was 'linke'. But I'll change to lenke then